CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
-# [0.0.1](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/tree/0.0.1) - 29th April, 2022
+# [0.0.1](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/tree/0.0.1)
 
 * Initial release.
+
+# [0.0.2](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/tree/0.0.2)
```

Comment:
ordering issue. latest version should be on top.

Reply:
Updated